public/index.html

```diff
@@ -1,18 +1,17 @@
 <!DOCTYPE html>
```

Comment:
All the changes made to this file were done by Prettier. We're not using tabs in this dojo, bruh.